### PR TITLE
Do not rebuild genesis block for testnet-prod

### PR DESCRIPTION
**Pull request by platfowner**

### CODE REVIEW

File: rebuild_genesis_blocks.sh [hunk 1]
```diff
@@ -14,7 +14,6 @@ rm blockchain-configs/testnet-dev/genesis_block.json.gz
 rm blockchain-configs/testnet-staging/genesis_block.json.gz
 rm blockchain-configs/testnet-sandbox/genesis_block.json.gz
 rm blockchain-configs/testnet-exp/genesis_block.json.gz
-rm blockchain-configs/testnet-prod/genesis_block.json.gz
```
Comment (liayoo) [inline rebuild_genesis_blocks.sh:17]: should we just remove mainnet-prod as well?

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: rebuild_genesis_blocks.sh (hunk 1 above)
Comment (liayoo) [reply rebuild_genesis_blocks.sh:17]: => already removed

Comment (platfowner) [reply rebuild_genesis_blocks.sh:17]: Right, that reminded me of updating multi-command files, which was just done. Thanks!

**Comment by platfowner** [comment]
Thanks for the fast review. Now merging.. @liayoo

File: rebuild_genesis_blocks.sh (hunk 1 above)
Comment (liayoo) [reply rebuild_genesis_blocks.sh:17]: Ahhh thank you!!

**Pull request merged**